perl/colorize_regex.pl

```diff
@@ -0,0 +1,1396 @@
+# SPDX-FileCopyrightText: 2025 ryoskzypu <ryoskzypu@proton.me>
+#
+# SPDX-License-Identifier: MIT-0
+#
+# License:
+#   MIT No Attribution
+#
+#   Copyright (c) 2025 ryoskzypu <ryoskzypu@proton.me>
+#
+#   Permission is hereby granted, free of charge, to any person obtaining a copy of this
```

Comment:
The complete license text will be available in the repository (in LICENSES/ directory), so having the license text in the script itself is optional.

You can safely remove it, but you can also keep it if you absolutely want it there (not a problem).